lib/active_model/serializer.rb

```diff
@@ -159,7 +173,10 @@ def initialize(object, options = {})
       @meta_key   = options[:meta_key]
       @scope      = options[:scope]
 
+      # SCOPE
       scope_name = options[:scope_name]
+      # Adds a class method with ':scope_name' that calls ':scope()'
+      # But isn't called anywhere
```

Comment:
I don't intend to include this comment in the final PR, but wanted to bring attention to scope/scope_name now being dead code, in case you'd like me to remove calling it from the controller and the builder.